.gitignore

```diff
@@ -1,6 +1,7 @@
 .DS_Store
 node_modules/
 wrangler-dist/
+miniflare-dist/
```

Comment:
Could we change this so that the outputs all go into `dist`? E.g.

```
dist
  wrangler
    cli.js
    cli.js.map
  miniflare
    cli.js
```

Reply:
Then we only need to ignore `dist` whatever else we decide to put in there.